### ISSUE TITLE
Improve basic metadata in HTML page source for accessibility and SEO

### ISSUE THREAD

**Issue by kintopp** [knaw-huc/textannoviz#65]
Add some semantics / structured tags to the page source, so that we get indexed properly by Google. For example, basic metadata such as title, description, language, hierarchical H1/H2.. etc.

**Comment by kintopp** [comment]
The <html lang> attribute is missing (accessibility)
No <h1>-<h6> elements are present in the Javascript enabled page source (accessibility).
No header, nav, main, footer, or aside HTML regions, or banner, navigation, main, or contentinfo roles are defined (accessibility)

**Comment by kintopp** [comment]
For now, I think we should include at least the following:

> Viewport Meta Tag: This tag is crucial for responsive design, which ensures
> that your website is accessible and usable on various devices, including mobile
> devices and tablets. This is particularly important for users with motor
> disabilities who may prefer to use a touch screen. The tag looks like this:
> `<meta name="viewport" content="width=device-width, initial-scale=1.0">`.
> 
> Language Attribute: While not a META tag, the lang attribute in the HTML tag
> is important for accessibility. It helps screen readers and translation tools
> understand the language of the page, which is crucial for people who use these
> assistive technologies. For example: `<html lang="en">`.
> 
> Meta Description: This tag provides a brief summary of the page's content.
> While primarily used for SEO, a clear and concise meta description can also
> benefit users with cognitive disabilities by giving them a quick overview of the
> page content.

**Comment by svandaalen** [comment]
First version has been implemented. Waiting on feedback from the Globalise team.

**Comment by kintopp** [comment]
Looks good, but I think there are still some syntax errors in the HTML:

```
13: Value of attribute “type” for element “<script>” is invalid; Content-type specification invalid.
13: Attribute “crossorigin” for element “<script>” requires a value but one was not specified.
59: Document type does not permit element “<div>” in content of element “<noscript>”.
```

**Comment by kintopp** [comment]
Looking into this some more, it looks like that the BBedit HTML validator is wrong about the errors in the comment above. <script type="module" crossorigin src="/assets/index-b1c75395.js"></script> is valid. HTML5 validator did catch that there is an opening but no closing <noscript> tag on the page (and also complained about trailing /s).
